_source/_posts/2022-02-28-shopping-app-with-node-minio.md

```diff
@@ -0,0 +1,706 @@
+body {
+    padding-left: 30px;
+    padding-top: 70px;
+}
+	
+/* navbar */
+.navbar-default {
+    background-color: #496572;
+    border-color: #496572;
+}
+/* title */
+.navbar-default .navbar-brand {
+    color: white;
+}
+.navbar-default .navbar-brand:hover,
+.navbar-default .navbar-brand:focus {
+    color: #5E5E5E;
+}
+/* link */
+.navbar-default .navbar-nav > li > a {
+    color: white;
+}
+.navbar-default .navbar-nav > li > a:hover,
+.navbar-default .navbar-nav > li > a:focus {
+    color: #333;
+}
+.navbar-default .navbar-nav > .active > a, 
+.navbar-default .navbar-nav > .active > a:hover, 
+.navbar-default .navbar-nav > .active > a:focus {
+    color: #555;
+    background-color: #E7E7E7;
+}
+.navbar-default .navbar-nav > .open > a, 
+.navbar-default .navbar-nav > .open > a:hover, 
+.navbar-default .navbar-nav > .open > a:focus {
+    color: #555;
+    background-color: #D5D5D5;
+}
+/* caret */
+.navbar-default .navbar-nav > .dropdown > a .caret {
+    border-top-color: #777;
+    border-bottom-color: #777;
+}
+.navbar-default .navbar-nav > .dropdown > a:hover .caret,
+.navbar-default .navbar-nav > .dropdown > a:focus .caret {
+    border-top-color: #333;
+    border-bottom-color: #333;
+}
+.navbar-default .navbar-nav > .open > a .caret, 
+.navbar-default .navbar-nav > .open > a:hover .caret, 
+.navbar-default .navbar-nav > .open > a:focus .caret {
+    border-top-color: #555;
+    border-bottom-color: #555;
+}
+/* mobile version */
+.navbar-default .navbar-toggle {
+    border-color: #DDD;
+}
+.navbar-default .navbar-toggle:hover,
+.navbar-default .navbar-toggle:focus {
+    background-color: #DDD;
+}
+.navbar-default .navbar-toggle .icon-bar {
+    background-color: #CCC;
+}
+@media (max-width: 767px) {
+    .navbar-default .navbar-nav .open .dropdown-menu > li > a {
+        color: #777;
+    }
+    .navbar-default .navbar-nav .open .dropdown-menu > li > a:hover,
+    .navbar-default .navbar-nav .open .dropdown-menu > li > a:focus {
+        color: #333;
+    }
+}	
+```
+
+## Run your application
+
+Run the following command to start the app server:
+
+```sh
+node index.js
+```
+
+To see the app, open a browser window and visit `http://localhost:3000`. Now log in to your application.
+
+
+## Do more with Node and MinIO
+
+We have set up a basic e-commerce app with Node.js, Express, and MinIO. We secured access to the store with the Okta Sign-In Widget. We also learned how to use the `flow` parameter to bootstrap Okta's Sign-In Widget for different pages within your application.
+
+If you'd like to learn more about building web apps with Node.js, you might want to check out these other great posts:
+
+- [Build Secure Node Authentication with Passport.js and OpenID Connect](/blog/2018/05/18/node-authentication-with-passport-and-oidc)
+- [Build User Registration with Node, React, and Okta](/blog/2018/02/06/build-user-registration-with-node-react-and-okta)
+- [Simple Node Authentication](/blog/2018/04/24/simple-node-authentication)
```

Comment:
Let's add one MinIO link to match the title. 
- [Build with MinIO JS SDK] (https://github.com/minio/minio-js)